config/settings.yml.example

```diff
@@ -56,3 +56,9 @@
 # Uncomment and modify if you want to change the log level
 # WARN, DEBUG, ERROR, FATAL, INFO, UNKNOWN
 #:log_level: ERROR
+
+# Log buffer size and extra buffer size (for errors)
+#:log_buffer: 2000
+#:log_buffer_errors: 1000
+# Log buffer threshold (debug messages are ignored by default)
+#:log_buffer_level: INFO
```

Comment:
The idea is simple - there are two buffers and the "tail" one only stores errors and fatals after they expire. This is not configurable. What you can configure is the input level filtering, by default it ignores debug messages.